packages/api/src/APIProvider.tsx

```diff
@@ -72,6 +72,15 @@ const getWebsocketInstance = (wss_url: string) => {
     return window.WSConnections[wss_url];
 };
 
+export const getWebsocket = () => {
+    const endpoint = getSocketURL();
+    const app_id = getAppId();
+    const language = localStorage.getItem('i18n_language');
+    const brand = 'deriv';
+    const wss_url = `wss://${endpoint}/websockets/v3?app_id=${app_id}&l=${language}&brand=${brand}`;
+    return window?.WSConnections?.[wss_url];
+};
```

Comment:
Do you want to also reuse this in `initializeDerivAPI` and remove the duplicated code from the lines 94 - 99?